bindings/gumjs/gumv8memory.cpp

```diff
@@ -647,6 +650,37 @@ gum_v8_memory_read (GumMemoryValueType type,
   }
 }
 
+GUMJS_DEFINE_FUNCTION (gum_v8_memory_read_volatile_array)
+{
+  gpointer address;
+  gssize length = -1;
+  gsize n_bytes_read = -1;
```

Comment:
(See earlier comment.)

For consistency with the style, where the C++ style differs in where variables are declared, `n_bytes_read` should be moved down to the line before it's used.